bucket/seafile-syncing-client.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://www.seafile.com/",
+    "description": "Seafile desktop syncing client",
+    "version": "7.0.9",
+    "license": "Apache-2.0",
+    "url": "https://download.seadrive.org/seafile-7.0.9-en.msi",
+    "hash": "3f837823c23403ad2d5aec29f81028aaf1e86055e2fa45a8b67fcf94f59ddade",
+    "extract_dir": "PFiles\\Seafile\\bin",
+    "env_set": {
+        "SEAFILE_DATA_DIR": "$dir\\data"
+    },
+    "shortcuts": [
+        [
+            "seafile-applet.exe",
+            "Seafile Desktop Syncing Client"
+        ]
+    ],
+    "persist": "data",
```

Comment:
What is this for?? The folder is not used

![image](https://user-images.githubusercontent.com/13260377/93771780-3ebccb00-fc1e-11ea-9aa4-25498776d8db.png)

Reply:
I'm not sure, but there's another environment variable which probably handles that directory as well. I'll implement it.

Reply:
Good find, haven't seen that before. It is creating a `C:\Users\<username>\ccnet` folder no matter what, but now the important `seafile.ini` is managed within the scoop directory.

Reply:
Then why do you nned the first environment variable?

Reply:
Apparently it's not needed if you set the `CCNET_CONF_DIR` env variable and the data directory in the `seafile.ini`. Anyways, the code states that it uses the environment variable at some point, even though it seems to ignore it here: https://github.com/haiwen/seafile-client/blob/afc17fc2a8022a82b90569f9136d21c47adb4843/src/configurator.cpp#L198-L203

I'd rather leave it be, the `SEAFILE_DATA_DIR` will not be used by any other application than Seafile though, so it won't hurt.

Reply:
Remove data dir environment variable. No need to flood users environment with junk variables.